website/static/games/ridgerider_games/minesweeper/index.html

```diff
@@ -12,22 +12,22 @@
             <div id="board"></div>
         </div>
         <div id="info">
-            <p id="info_timer">Timer - 0:0</p>
-            <p id="info_flags">Flags used - 0</p>
-        </div>
-        <div id="settings">
-            <h6>Squares adjacent to bombs glow red the more bombs that are near</h6>
-            <div id="setting_danger">
-                <button id="setting_danger_inner" onclick="setting('danger')"></button>
+            <div id="info_timer">
+                <p id="info_timer_text">0:00</p>
             </div>
-            <h6>Flagging mode</h6>
-            <div id="setting_flag">
-                <button id="setting_flag_inner" onclick="setting('flag')"></button>
+            <div id="info_flags" onclick="setting('flag')">
+                <img id="info_flags_img" src="just_flag.png">
+                <p id="info_flags_text">0/30</p>
```

Comment:
It's best practice to use classes instead of ids to style html elements. That way you can reuse them for different elements.  For example, you have two IDs: `#info_flags_text and #info_timer_text`. Instead, you could have one CSS class that's descriptive like `.no-user-select` since that's what the CSS does.